### ISSUE TITLE
PEM12

### ISSUE THREAD

**Issue by maridansoft** [yandex-money/yandex-money-sdk-php#17]
Do you support in this SDK the SSL sertificate signing which yandex implemented lately?

**Comment by raymank26** [comment]
Yes I am. Do you have any problems? You can check CA bundle by using OpenSSL s_client like this: `openssl s_client -connect money.yandex.ru:443 -CAfile cacert.pem -crlf -verify 9`. The last output line should be `Verify return code: 0 (ok)`

**Comment by maridansoft** [comment]
The problem is that we cannot figure out where to put the pem12 which we get from yandex. where to install it ?
